virtual-list.js

```diff
@@ -361,20 +377,21 @@ export const RepeatsAndScrolls = Superclass => class extends Repeats
       this._notifyStable();
     }
   }
-  /**
-   * TODO(valdrin) use ResizeObserver to compute container visibility.
-   * @private
-   */
-  get _isContainerVisible() {
-    const bounds = this._containerElement.getBoundingClientRect();
-    return Boolean(bounds.width || bounds.height || bounds.top || bounds.left);
-  }
   /**
    * @protected
    */
   _shouldRender() {
-    return Boolean(
-        super._shouldRender() && this._layout && this._isContainerVisible);
+    if (!super._shouldRender() || !this._layout) {
+      return false;
+    }
+    // NOTE: we're about to render, but the ResizeObserver didn't execute yet.
+    // Since we want to keep rAF timing, we compute _containerSize now.
+    // Would be nice to have a way to flush ResizeObservers
```

Comment:
@domenic do you know of some way to force ResizeObserver to provide size?
If we wait for ResizeObserver timing, we won't render within a rAF the first time, e.g.
```js
vlist.newChild = () => document.createElement('button');
vlist.totalItems = 10;
requestAnimationFrame(() => {
  // expected to be 11 (10 buttons + 1 sizer div)
  console.log(vlist.childElementCount); 
});

Reply:
I think manually asking the first time seems OK, but @atotic may have ideas.